op-node/node/node.go

```diff
@@ -207,12 +209,26 @@ func (n *OpNode) initL1(ctx context.Context, cfg *config.Config) error {
 		return fmt.Errorf("failed to validate the L1 config: %w", err)
 	}
 
+	return nil
+}
+
+func (n *OpNode) initL1Handlers(cfg *config.Config) error {
+	if n.l2Driver == nil {
+		return errors.New("l2 driver must be initialized")
+	}
 	emitter := n.eventSys.Register("l1-signals", nil)
 	onL1Head := func(ctx context.Context, sig eth.L1BlockRef) {
-		emitter.Emit(ctx, status.L1UnsafeEvent{L1Unsafe: sig})
+		// TODO(#16917) Remove Event System Refactor Comments
+		//  L1UnsafeEvent fan out is updated to procedural method calls
+		if n.cfg.Tracer != nil {
+			n.cfg.Tracer.OnNewL1Head(ctx, sig)
+		}
+		n.l2Driver.L1Tracker.OnL1Unsafe(sig)
```

Comment:
Previously, was one single event emission being consumed by multple handlers at once?

Reply:
Yes. `L1UnsafeEvent` was broadcasted and consumed by multiple handlers. `L1Tracker`, `SyncDeriver`, `StatusTracker` were the consumers. `Tracer` also consumed when enabled.